src/app/armory/Armory.tsx

```diff
@@ -14,9 +14,10 @@ import {
   useSocketOverrides,
 } from 'app/inventory/store/override-sockets';
 import { getEvent, getSeason } from 'app/inventory/store/season';
+import { AmmoIcon } from 'app/item-popup/AmmoIcon';
 import BreakerType from 'app/item-popup/BreakerType';
 import EmblemPreview from 'app/item-popup/EmblemPreview';
-import { AmmoIcon, ItemTypeName } from 'app/item-popup/ItemPopupHeader';
+import {} from 'app/item-popup/ItemPopupHeader';
```

Comment:
```suggestion
```

here and in some other files below

Reply:
Weird, I would think VSCode would automatically remove those.